js/shared.js

```diff
@@ -139,14 +139,19 @@ export const getMyData = async () => {
 
     const idToken = await getIdToken();
     const response = await fetch(`${api}?api=getUserProfile`, {
-      headers: {
-        Authorization: 'Bearer ' + idToken,
-      },
+        headers: {
+            Authorization: 'Bearer ' + idToken,
+        },
     });
 
     return response.json();
 };
 
+export const hasUserData = (response) => {
+
+    return response.code === 200 && Object.keys(response.data) > 0;
```

Comment:
Is this the intended comparison? I think we may be looking for the `.length` operator:
```javascript
return response.code === 200 && Object.keys(response.data).length > 0;
```

Reply:
Great catch!